Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionActionCocoa.mm

```diff
@@ -736,14 +737,14 @@ - (void)_otherPopoverWillShow:(NSNotification *)notification
 
 #if ENABLE(WK_WEB_EXTENSIONS_ICON_VARIANTS)
         if (m_customIconVariants) {
-            result = extensionContext()->extension().bestImageForIconVariants(m_customIconVariants.get(), idealSize, [&](auto *error) {
-                extensionContext()->recordError(error);
+            result = extensionContext()->extension().bestImageForIconVariants(m_customIconVariants.get(), idealSize, [&](Ref<API::Error> error) {
+                extensionContext()->recordError(::WebKit::wrapper(error));
```

Comment:
```suggestion
                extensionContext()->recordError(wrapper(error));
```

Reply:
As mentioned above, just using wrapper calls WebExtensionAction::wrapper instead of the wrapper we want